demos/src/data.json

```diff
@@ -11,6 +11,7 @@
     	"withGoogleplus": true,
     	"withReddit": true,
     	"withPinterest": true,
-    	"withLink": true
+    	"withLink": true,
+        "withWhatsapp": true
```

Comment:
Nicer to put it in the same order as the elements appear in on the spec markup (main.mustache)